### PR TITLE
Add rotary switches

### PR DESCRIPTION
**Pull request by Rmano**
I couldn't resist adding these, even if 0.9.1 was deemed to be a bug-fix release... 

Description here:
[rotary.pdf](https://github.com/circuitikz/circuitikz/files/3232077/rotary.pdf)

### CODE REVIEW

File: doc/circuitikzmanual.tex [hunk 1]
```diff
@@ -2438,6 +2447,135 @@ \subsubsection{Cute switches customization}
...
+\draw[blue] (A-out 2.n) -- ++(0,0.5) -| (B-out 1.n);
+\draw (B.out 3) -- ++(1,0) node[rotary switch -=5 in 90 wiper 15, anchor=in](C){};
+\draw (C.out 3) -- ++(1,0) node[rotary switch ->, xscale=-1, anchor=out 3](D){};
+\draw[green, dashed] (B.mid) -- ++(-.5,-1) -| (C.mid);
+\end{circuitikz}
+\end{LTXexample}
+
+\paragraph{Rotary switch anchors}
+
+Rotary switches anchors are basically the same as the cute switches, included the access (with the \texttt{\textsl{<node name>}-<anchor name>} notation) to the internal connection nodes. The geographical anchors works as expected, marking the limits of the component.
```
Comment (asinghvi17) [inline doc/circuitikzmanual.tex]:
```suggestion
Rotary switches anchors are basically the same as the cute switches, including access (with the \texttt{\textsl{<node name>}-<anchor name>} notation) to the internal connection nodes. The geographical anchors work as expected, marking the limits of the component.
```

File: doc/circuitikzmanual.tex [hunk 2]
```diff
@@ -2438,6 +2447,135 @@ \subsubsection{Cute switches customization}
...
+\begin{LTXexample}[varwidth=true, basicstyle=\small\ttfamily]
+\begin{circuitikz}
+\draw (0,0) node[rotary switch=13 in 120 wiper 0](S){};
+\foreach \i in {1,...,13} % requires "calc"
+    \path ($(S.aout \i)!1ex!(S.sqout \i)$)
+        node[font=\tiny\color{red}]{\i};
+\end{circuitikz}
+\end{LTXexample}
+
+Finally, notice that the value of width for the rotary switches is taken from the one for the ``cute switches'' which in turn is taken from the width of standard spdt, so that they match (notice that the ``center'' anchor is better centered in the rotary switch, so you have to explicitly align them).
```
Comment (asinghvi17) [inline doc/circuitikzmanual.tex]: `spdt`?

File: doc/circuitikzmanual.tex [hunk 3]
```diff
@@ -2438,6 +2447,135 @@ \subsubsection{Cute switches customization}
...
+        anchor=in](R){};
+    \draw (0,-1) node[cute spdt up, anchor=in](C){};
+    \draw (0,-2) node[color=blue, rotary switch=3 in 35 wiper 30,
+        anchor=in](R){};
+\end{circuitikz}
+\end{LTXexample}
+
+\paragraph{Rotary switch customization}
+
+A part from the basic customization seen above (number of channels, etc.) you can change, as in the cute switches, the shape used by the connection points with the parameter \texttt{multipoles/rotary/shape}, and the thickness of the wiper with \texttt{multipoles/rotary/thickness}. The optional arrow has thickness equal to the standard bipole thickness \texttt{bipoles/thickness} (default 2).
```
Comment (asinghvi17) [inline doc/circuitikzmanual.tex]:
```suggestion
Apart from the basic customization seen above (number of channels, etc.) you can change, as in the cute switches, the shape used by the connection points with the parameter \texttt{multipoles/rotary/shape}, and the thickness of the wiper with \texttt{multipoles/rotary/thickness}. The optional arrow has thickness equal to the standard bipole thickness \texttt{bipoles/thickness} (default 2).
```

File: doc/circuitikzmanual.tex (hunk 2 above)
Comment (Rmano) [reply doc/circuitikzmanual.tex]: yes, the "traditional" single-pole double-throw switch.

Comment (asinghvi17) [reply doc/circuitikzmanual.tex]: Makes sense, thanks!

**Pull request merged**